ntbs-service/Pages/Shared/_NotificationLayout.cshtml

```diff
@@ -19,7 +19,7 @@
             break;
     }
 
-    ViewData["IsOverviewPage"] = currentPage == string.Empty;
+    ViewData["IsOverviewPage"] = currentPage == string.Empty || currentPage == @Model.NotificationId.ToString();

```

Comment:
@conorsheehansw your fix will supersede this actually, so you can ignore this line :)